pipelines/toast_ground_sim.py

```diff
@@ -1896,129 +470,136 @@ def main():
     # Expand boresight quaternions into detector pointing weights and
     # pixel numbers
 
-    expand_pointing(args, comm, data, mem_counter)
+    expand_pointing(args, comm, data)
+
+    # Purge the pointing if we are NOT going to export the
+    # data to a TIDAS volume
+    if (args.tidas is None) and (args.spt3g is None):
+        for ob in data.obs:
+            tod = ob["tod"]
+            tod.free_radec_quats()
 
     # Prepare auxiliary information for distributed map objects
 
     _, localsm, subnpix = get_submaps(args, comm, data)
 
-    if args.input_pysm_model:
+    if args.pysm_model:
+        focalplanes = [s[3] for s in schedules]
```

Comment:
using indices is kind of error prone, can we create a named tuple or simple class (dataclasses unfortunately are python 3.7 only)?